docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -161,6 +161,12 @@ signature verified for data2: true
 Fields in signature: 256
 ```
 
+where:
+
+- `Fields in private key: 255` is a bit field element of the scalar field of the curve
```

Comment:
I don't think "bit field" is a good term, to me it's confusing. So, to me, this doesn't clearly explain why a private key has 255 field elements, and I'm also questioning whether this is a good place to explain that in the first place.

Reply:
@mitschabaude shall we forgo comments altogether? Or do you have a better way to describe?